cmd/utils/executor/executor.go

```diff
@@ -61,7 +61,7 @@ func NewExecutor(output string) *Executor {
 // Execute executes the specified command adding `env` to the execution environment
 func (e *Executor) Execute(cmdInfo model.DeployCommand, env []string) error {
 
-	cmd := exec.Command("bash", "-c", cmdInfo.Command)
+	cmd := exec.Command(cmdInfo.Command)
```

Comment:
This is going to change how commands are executed. Could we set this as a flag, announce deprecation, then do the full switch in a version or two? (Or announce it as a breaking change).

Reply:
@rberrelleza what do you mean by setting this as a flag? the new behavior under the flag? If we do that, we'll be throwing a warning every time a user runs `okteto deploy/destroy` to use a file flag that is going to be deprecated.

If we set the old behavior under the flag, are we going to release a flag that is already deprecated? I prefer this idea and show the users a way of getting back the old behavior(via flag). We should also track how many people are having errors vs previous versions and see if we are having more errors after this change, WDYT?

Reply:
Setting as default the new behavior could break customers' scripts/pipelines which ideally we should try to avoid unless we announce it as a breaking change and update the versions accordingly. 

If I understand correctly one of the issues, does this prevent the `okteto deploy` command to work on Windows? If so, we might enforce the new behavior just when the OS is windows. I mean, if for those cases it's not working, we won't break anything changing the behavior. For the rest of OS we can include a flag **defaulting to the old behavior**

Reply:
@ifbyol The problem is that windows bash runs on an auxiliary operating system, which does not run on the same path or with the same user. Being an auxiliary operating system it does not correctly pair absolute paths.

We can activate it only for Windows, although I would like that there is no difference between different operating systems, since it is more difficult to maintain it.

Reply:
@jLopezbarb I know, but one of our main goals is stability in the CLI, so a breaking change I don't think is a good option right now.

My point is, if in Windows this was never working as expected, doing this breaking change only for Windows would fix the issue. Now, if this wasn't working properly on some situations, then, maybe we have to rethink it because we could be broken other scenarios

Reply:
What if we add a feature flag to remove the `bach -c`? In this case we wouldn't do a breaking change and we'd also allow for the "correct" behavior with a flag like `--execute-without-bash`. We could incentivize the use of this flag and postpone it as the default behavior in some future big release

WDYT @ifbyol @jLopezbarb ?